### DISCUSSION TITLE
sub-workflow

### DISCUSSION THREAD

**Discussion by weizhu365** [ropensci/targets#1396] [General]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

Users may define a function for each tar_target.  A complicate function can be written as a workflow itself.  Is there a way to make sub-workflow using R targets?

Thanks,

Wei Zhu

**Comment by wlandau** [comment]
You could have different projects: https://books.ropensci.org/targets/projects.html. Also c.f. https://github.com/ropensci/targets/discussions/1350

**Comment by weizhu365** [comment]
You are right.  That is a good idea to use "share projects".  

Thank you very much!
